karma.conf.js

```diff
@@ -5,7 +5,7 @@ delete webpackConfig.output;
 module.exports = function(config) {
   config.set({
     frameworks: ['mocha', 'sinon-chai'],
-    browsers: ['Firefox'],
+    browsers: ['ChromeHeadless'],
```

Comment:
Rationale?

Reply:
I'm having a hard time running browser tests in Github Actions. `karma` was failing when trying to launch Firefox and I saw a SO thread recommending to use `ChromeHeadless` instead for CI.

Reply:
Seems karma can only launch headless versions of browsers in Github Actions. I've updated it to FirefoxHeadless and it works well too.

Reply:
Makes sense, thanks for resolving this! Do you think we should run it on both FF and Chrome?

Reply:
Sure, done in https://github.com/stellar/js-stellar-base/pull/457/commits/9623cb5742ca1cafb6c9f3242420967df4a2764e.